src/rt/sched/cown.h

```diff
@@ -534,19 +527,21 @@ namespace verona::rt
                            << next << ", index " << body->index << std::endl;
 
         {
-          // Hold an epoch so that a scheduled `next` cown is not collected by
-          // the time that we must set it unmutable.
-          Epoch e(alloc);
+          if (make_unmutable)
+            next->unmute(true);
+
```

Comment:
If we sleep at this point for a long time, is it possible for the unmutable state to be lost before we send the message?

Reply:
1. I'll update to remove the term "unmutable" from the code.
2. It is possible that the the required cowns will lower their priority before they are acquired. But it's ok for priority to be raised when it is not strictly necessary, as long as it's not too frequent.

Reply:
But can't the priority go back down before the send, and then we reach a deadlock?

Reply:
The priority of the running cown will not go down. Priority can only be raised from another scheduler thread, not lowered

Reply:
But `next` is not running on this scheduler thread yet, it could be running on another scheduler thread, that processes some messages and empties its queue, and puts it into `Normal` priority due to processing its message queue be empty, or the token pops out a couple of times, and then a behaviour on that cown  sends something to an overloaded cown, and goes to `Low` priority, and then this message send occurs?

Reply:
You are correct. This is an issue. I'll go back to holding an epoch on each attempt to fast-send and raising priority if interrupted.